ADALiOS/ADALiOS/ADAuthenticationResult.h

```diff
@@ -54,14 +54,15 @@ typedef enum
  case of error.*/
 @property (readonly) NSString* accessToken;
 
+/*! A valid access token, if the results indicates success. The property is
+ calculated from the tokenCacheStoreItem one. The property is nil, in
+ case of error.*/
+@property (readonly) NSString* idToken;
```

Comment:
why do we have idToken as a property? You can either get an access token OR an id token from the acquireToken request. Please refer to .NET implementation for guidelines.